src/pages/panel/views/simple.js

```diff
@@ -12,60 +12,70 @@
 import { html, define, store, router } from 'hybrids';
 import { t } from '/vendor/@whotracksme/ui/src/i18n.js';
 
-import '/vendor/@whotracksme/ui/src/components/wtm-stats.js';
 import {
   externalLink,
   chevronRight,
 } from '/vendor/@whotracksme/ui/src/components/icons.js';
 
-import { rulesetIds } from '../utils/rulesets.js';
 import sites from '../../../rule_resources/sites.json';
 
-import Stats from '../store/stats.js';
-import Settings from '../store/settings.js';
+import Stats from '/store/stats.js';
```

Comment:
why absolute import?

Reply:
Cosmetics, just to avoid `../../../` paths, when we use something from general module (`store`, `ui`, `rule_resources`, etc) we can make a rule to use absolute path.